routes/router.js

```diff
@@ -1,13 +1,15 @@
 var rootRouter = require("express").Router();
 
 var demoController = require('../controllers/demoController');
+var messageController = require('../controllers/messageController');
 
-module.exports = function(app, contextRoot){
+module.exports = function(app, contextRoot) {
     app.use(contextRoot, rootRouter);
 
-	rootRouter.get('/demo', demoController.demo);
-	rootRouter.post('/addDemo', demoController.addDemo);
+    rootRouter.get('/demo', demoController.demo);
+    rootRouter.post('/addDemo', demoController.addDemo);
 
     //Please make any business router under the rootRouter, so that it will be easy for contextRoot config.
-
-};
\ No newline at end of file
+    rootRouter.get('/message', messageController.message);
```

Comment:
It's better to have your own router, say messageRouter, set to rootRouter("/message"). Then you may not need to touch the file router.js every time you change message API.